src/exporters/sieve.rs

```diff
@@ -5,9 +5,11 @@ use std::io::{Error, ErrorKind, Result, Write};
 use crate::exporters::Export;
 use crate::Operation;
 
+pub struct IR0;
 pub struct IR1;
 
 impl Export<bool> for IR1 {
+    //{{{
```

Comment:
What's the with the `{{{` comment block here and below?

Reply:
oops, they're vim folds that i forgot.